glean-core/src/database/mod.rs

```diff
@@ -206,11 +220,36 @@ impl Database {
         store.put(&mut writer, final_key, &value).unwrap();
         let _ = writer.commit();
     }
+
+    /// Clears a storage (only Ping Lifetime).
+    pub fn clear_ping_lifetime_storage(&self, storage_name: &str) {
```

Comment:
Should we make this generic by passing in the lifetime enum?

Reply:
I'd rather not do this for now, since all we want is clearing the Ping lifetime. Unless there's some other use-case I've missed?

Reply:
We will eventually need a way to delete the `user` storage as well, but I think we can tackle that when we get to it.

Reply:
Good point!